backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -643,11 +643,34 @@ export class ExperimentService {
 
         experimentDoc.experimentSegmentInclusion = oldExperiment.experimentSegmentInclusion;
         experimentDoc.experimentSegmentExclusion = oldExperiment.experimentSegmentExclusion;
+
+        const currentIncludeData = experimentSegmentInclusion.segment
+          ? {
+              ...experimentSegmentInclusion.segment,
+              userIds: experimentSegmentInclusion.segment.individualForSegment.map((x) => x.userId),
```

Comment:
can we please name things what they are instead of using `x`? thank you

Reply:
Code updated